src/frontend/src/common/components/library/data_subview/components/createTreeModal/index.tsx

```diff
@@ -59,19 +141,111 @@ export const CreateTreeModal = ({
+                        Tree names must be no longer than 128 characters.
+                      </InstructionsNotSemiBold>
+                    </span>,
+                  ]}
+                />
+              )}
+              <StyledTextField
+                fullWidth
+                error={isTreeNameTooLong}
+                id="outlined-basic"
+                variant="outlined"
+                value={treeName}
+                onChange={(e) => setTreeName(e.target.value)}
+              />
+              {isTreeNameTooLong && (
+                <TreeNameTooLongAlert>
+                  <StyledErrorOutlinedIcon />
+                  <TextFieldAlert>
+                    Name exceeds the 128 character limit.
+                  </TextFieldAlert>
+                </TreeNameTooLongAlert>
+              )}
+            </TreeNameSection>
+            <TreeTypeSection>
+              <TreeNameInfoWrapper>
+                <FieldTitle>
+                  Include publicly-available samples from:{" "}
+                </FieldTitle>
+                <StyledTooltip
+                  arrow
+                  title={TREE_TYPE_TOOLTIP_TEXT}
+                  placement="top"
+                >
+                  <StyledInfoOutlinedIcon />
+                </StyledTooltip>
+              </TreeNameInfoWrapper>
+              <RadioGroup
+                value={treeType}
+                onChange={(e) => setTreeType(e.target.value)}
+              >
+                <StyledFormControlLabel
+                  value="contextual"
+                  control={<StyledRadio />}
+                  label={<RadioLabelContextual selected={isContextual} />}
+                />
+                <StyledFormControlLabel
+                  value="local"
+                  control={<StyledRadio />}
+                  label={<RadioLabelLocal selected={isLocal} />}
+                />
+              </RadioGroup>
+            </TreeTypeSection>
+            {failedSamples.length > 0 && (
+              <Alert severity="warning">
+                <InstructionsSemiBold>
+                  {" "}
+                  {failedSamples.length} Selected Sample
+                  {failedSamples.length > 1 && "s"} wont be included in your
+                  tree{" "}
+                </InstructionsSemiBold>
+                <InstructionsNotSemiBold>
+                  because they failed genome recovery.
+                </InstructionsNotSemiBold>
+              </Alert>
+            )}
+            <StyledButton
+              color="primary"
+              variant="contained"
+              isRounded
+              disabled={isTreeBuildDisabled}
+              type="submit"
+              value="Submit"
+            >
+              Create Tree
+            </StyledButton>
+          </form>
+          {isCreateTreeButtonPressed && (
+            <CreateTreeInfo>
+              Creating a new tree can take up to X hours. We will notify you
```

Comment:
How many hours should we tell the user here?

Reply:
the designs say X hours so i was adding that as a placeholder here until we got confirmation from design/ product. I'll check with slack